src/igvfd/searches/configs/measurement_set.py

```diff
@@ -9,122 +9,208 @@ def measurement_set():
         'facets': {
             'assay_term.assay_slims': {
                 'title': 'Assay',
+                'category': 'Measurement Set Details',
+                'description': 'High-level assay classification based on OBI assay slim terms.',
                 'type': 'hierarchical',
                 'subfacets': [
-                    {
-                        'field': 'assay_term.term_name',
-                        'title': 'Assay type',
-                    }
+                    {'field': 'assay_term.term_name', 'title': 'Assay type'}
                 ]
             },
             'preferred_assay_titles': {
-                'title': 'Preferred Assay Titles'
+                'title': 'Preferred Assay Titles',
+                'category': 'Measurement Set Details',
+                'description': 'Title of assays that generated the measurement set.'
             },
             'control_types': {
-                'title': 'Control Types'
+                'title': 'Control Types',
+                'category': 'Measurement Set Details',
+                'description': 'The type of control for the measurement set, if applicable.',
+                'optional': True
             },
-            'functional_assay_mechanisms.term_name': {
-                'title': 'Measured Mechanisms'
+            'targeted_genes.symbol': {
+                'title': 'Readout Gene',
+                'category': 'Measurement Set Details',
+                'description': 'A list of genes targeted for binding sites or used for sorting by expression.',
+                'optional': True
             },
             'auxiliary_sets.file_set_type': {
-                'title': 'Auxiliary Data'
+                'title': 'Auxiliary Data',
+                'category': 'Measurement Set Details',
+                'description': 'Additional file sets that supplement or support the primary measurement set.',
+                'optional': True
             },
-            'files.sequencing_platform.term_name': {
-                'title': 'Sequencing Platform'
-            },
-            'library_preparation_kit': {
-                'title': 'Library Preparation Kit'
+            'functional_assay_mechanisms.term_name': {
+                'title': 'Measured Mechanisms',
+                'category': 'Measurement Set Details',
+                'description': 'Ontological term that describes the biological processes measured by this functional assay.',
```

Comment:
"The biological processes measured by this functional assay. For example, a VAMP-seq (MultiSTEP) assay measures the effects of variants on protein carboxylation and secretion processes."